packages/api/src/command/webhook/devices.ts

```diff
@@ -35,8 +35,8 @@ export const sendProviderConnected = async (
   let webhookRequest;
   try {
     const { id: userId, cxId } = connectedUser;
-    const providers = connectedUser?.providerMap ? Object.keys(connectedUser.providerMap) : null;
-
+    const providers = connectedUser?.providerMap ? Object.keys(connectedUser.providerMap) : [];
+    providers.push(provider);
```

Comment:
I think this doesn't address the original problem, which is the `connectedUser` not having an updated `providerMap`.

Looking at the code, it seems we need to add the line selected below to `updateProviderData`, it should fix the problem:

<img width="680" alt="image" src="https://github.com/metriport/metriport/assets/2132564/ed3acb6b-1ed3-4ff9-a9fe-f4f6a217d880">

Reply:
Unfortunately, we can't return the result of `ConnectedUser.update` b/c it returns the number of updated records, not the actual entity.

Reply:
Could try to move it to `.save()`, just need to test it well since we had issues with those "active record" functions from Sequelize - [context](https://metriport.slack.com/archives/C04DMKE9DME/p1686779391180389):

```ts
  connectedUser.providerMap = newProviderMap;
  return connectedUser.save();
```